README.md

```diff
@@ -9,13 +9,15 @@ A starter repository showing how to build a blog with the [Eleventy](https://git
 * [Netlify](https://eleventy-base-blog.netlify.com/)
 * [GitHub Pages](https://11ty.github.io/eleventy-base-blog/)
 * [Remix on Glitch](https://glitch.com/~11ty-eleventy-base-blog)
+* [Azure Static Web Apps](https://aka.ms/11ty-swa)
 
 ## Deploy this to your own site
 
 These builders are amazing—try them out to get your own Eleventy site in a few clicks!
 
 * [Get your own Eleventy web site on Netlify](https://app.netlify.com/start/deploy?repository=https://github.com/11ty/eleventy-base-blog)
 * [Get your own Eleventy web site on Vercel](https://vercel.com/import/project?template=11ty%2Feleventy-base-blog)
+* [Get your own Eleventy web site on Azure Static Web Apps](https://github.com/manekinekko/11ty-blog-swa)
```

Comment:
And this link points to the app URL to deploy this `eleventy-base-blog` repo on the service.